seatunnel-config/seatunnel-config-shade/src/test/java/org/apache/seatunnel/config/utils/FileUtils.java

```diff
@@ -27,16 +27,13 @@ private FileUtils() {
 
     // get file from classpath, resources folder
     public static File getFileFromResources(String fileName) {
-
-        ClassLoader classLoader = FileUtils.class.getClassLoader();
-
+        ClassLoader classLoader = Thread.currentThread().getContextClassLoader();
```

Comment:
I debug at my laptop, the classloader is the same with `FileUtils.class.getClassLoader`.
And I find there is a better way to solve these path issue in window. https://stackoverflow.com/questions/6164448/convert-url-to-normal-windows-filename-java
```
URL resource = classLoader.getResource(fileName);
Paths.get(resource.toURI()).toFile();
```